content/pt-br/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -0,0 +1,356 @@
+especificação de recurso de credencial GMSA. A seguir mostra o serviço de conta 
+padrão vinculado a `cluster role` `webapp1-role`, para usar a especificação 
+de recurso da credencial `gmsa-WebApp1` criada acima.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: allow-default-svc-account-read-on-gmsa-WebApp1
+  namespace: default
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: webapp1-role
+  apiGroup: rbac.authorization.k8s.io
+```
+
+## Configure a especificação de recurso de credencial GMSA em uma especificação de Pod
+
+O campo de especificação do Pod `securityContext.windowsOptions.gmsaCredentialSpecName`,  
+é usado para especificar referências para recursos customizados, em especificações 
+de certificado GMSA apropriadas em especificações do Pod. 
+Isto configura todos contêineres na especificação do Pod para usar a especificação GMSA. 
+Uma amostra da especificação do Pod com anotação populada para referir-se a `gmsa-WebApp1`:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    run: with-creds
+  name: with-creds
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: with-creds
+  template:
+    metadata:
+      labels:
+        run: with-creds
+    spec:
+      securityContext:
+        windowsOptions:
+          gmsaCredentialSpecName: gmsa-webapp1
+      containers:
+      - image: mcr.microsoft.com/windows/servercore/iis:windowsservercore-ltsc2019
+        imagePullPolicy: Always
+        name: iis
+      nodeSelector:
+        kubernetes.io/os: windows
+```
+
+Contêineres individuais em uma especificação de Pod, podem também especificar 
+a apropriada especificação de credencial GMSA, usando um campo por contêiner 
+`securityContext.windowsOptions.gmsaCredentialSpecName`. Por exemplo:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    run: with-creds
+  name: with-creds
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: with-creds
+  template:
+    metadata:
+      labels:
+        run: with-creds
+    spec:
+      containers:
+      - image: mcr.microsoft.com/windows/servercore/iis:windowsservercore-ltsc2019
+        imagePullPolicy: Always
+        name: iis
+        securityContext:
+          windowsOptions:
+            gmsaCredentialSpecName: gmsa-Webapp1
+      nodeSelector:
+        kubernetes.io/os: windows
+```
+
+Assim que as especificações do Pod com os campos GMSA preenchidos 
+(como descrito acima) são aplicadas em um cluster, ocorre a seguinte sequência de eventos:
+
+1. O webhook mutante resolve e expande todas as referências aos recursos de 
+especificações de credenciais GMSA para o conteúdo das especificações de credenciais GMSA.
+
+1. O webhook validador garante que a conta de serviço associada ao Pod, seja autorizada 
+para o `uso` verbal na especificação GMSA especificada.
```

Comment:
Few bits here:
* I think using "webhook de mutação" and "webhook de validação" would be better.
* `use` should not be translated in this context. Suggestion: "seja autorizada para o verbo `use`".